proto/dydxprotocol/clob/query.proto

```diff
@@ -41,6 +42,16 @@ service Query {
       returns (QueryEquityTierLimitConfigurationResponse) {
     option (google.api.http).get = "/dydxprotocol/clob/equity_tier";
   }
+
+  // Queries all stateful orders.
+  rpc AllStatefulOrders(QueryAllStatefulOrdersRequest)
+      returns (QueryAllStatefulOrdersResponse) {
+    option (google.api.http).get = "/dydxprotocol/clob/orders/stateful";
+  }
+
+  // Queries the count of all stateful orders.
+  rpc StatefulOrderCount(QueryStatefulOrderCountRequest)
```

Comment:
no http transcoding here because the request is uniquely identified by the subaccount. when I added an http path that selected on the subaccount message, go proto-gen compiled without issues, but telescope barfed.

Reply:
for my understanding - what will this endpoint be used for?

this is not something that is in the consensus state

Reply:
See Jonathan's comment [here](https://dydx-team.slack.com/archives/C03SLFHC3L7/p1697746756832109?thread_ts=1697738956.526649&cid=C03SLFHC3L7)